llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -5821,7 +5821,8 @@ void LoopVectorizationCostModel::collectInstsToScalarize(ElementCount VF) {
         // invalid scalarization costs.
         // Do not apply discount logic if hacked cost is needed
         // for emulated masked memrefs.
```

Comment:
Looks like the comment still needs updating, to include the new condition.

Reply:
Sorry for the delay! See #91120.